upstream/kueue/e2e-test-ocp.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+# e2e-test-ocp.sh
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export OC=$(which oc) # OpenShift CLI
+export GINKGO=$(pwd)/bin/ginkgo
+SOURCE_DIR="$(cd "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)"
+ROOT_DIR="$SOURCE_DIR/.."
+DEFAULT_NAMESPACE="openshift-kueue-operator"
```

Comment:
I don't think you need this. Setting `KUEUE_NAMESPACE` will override that value.

Reply:
You are correct. Pushed a fix for that.